snippets/olmv0-rn-np-support.adoc

```diff
@@ -15,7 +15,7 @@ In {product-title} {product-version}.{olm-np-z-stream}, {olmv0-first} supports t
 
 [TIP]
 ====
-An insights rule on `console.redhat.com` alerts you to update to {product-title} {product-version}.{olm-np-z-stream} or later, if the currently installed version of OLM does not support tailored network policies.
+If your current version of OLM does not support tailored network policies, a notification is displayed on console.redhat.com and the web console. Update to {product-title} {product-version}.{olm-np-z-stream} or later to enable OLM support for tailored network policies.
```

Comment:
If your current version of OLM does not support tailored network policies, a notification is displayed on console.redhat.com and the OpenShift web console of the (impacted or affected) cluster..